test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/RelativePath.java

```diff
@@ -54,6 +54,16 @@ private static void doTest(String topArchiveName) throws Exception {
         int idx = appJar.lastIndexOf(File.separator);
         String jarName = appJar.substring(idx + 1);
         String jarDir = appJar.substring(0, idx);
+
+        // Create CDS Archive
+        dump(topArchiveName, "-Xlog:cds",
+            "-Xlog:cds+dynamic=debug",
+            "-cp", "." + File.separator + "hello.jar" + File.pathSeparator + appJar2,
```

Comment:
For dumping archive, I think it's better not to use relative path as follows:
`"-cp", appJar + File.pathSeparator + appJar2,`

This test is for validating using relative path during runtime works while relative path wasn't used during dump time.